packages/nexus-sdk/src/View/index.ts

```diff
@@ -132,9 +122,17 @@ const View = (
       viewId: string,
     ): Promise<Statistics> =>
       httpGet({
-        path: `${
-          context.uri
-        }/views/${orgLabel}/${projectLabel}/${viewId}/statistics`,
+        path: `${context.uri}/views/${orgLabel}/${projectLabel}/${viewId}/statistics`,
+      }),
+    pollStatistics: (
+      orgLabel: string,
+      projectLabel: string,
+      viewId: string,
+      options?: { pollTime: number },
```

Comment:
Since it's milliseconds (I assume), it should be made explicit with a name like `pollTimeMs`, otherwise some people will think it's in seconds, or will have to take time to check which one it is.

Reply:
If it's an interval, I would even name it `pollIntervalMs`

Reply:
Fair point !

Reply:
I hope your ready for the PR now ;-)

Reply:
fixed in 9bc2295e19832275c61357eebea6c0df01989756